iChef-WebUi/src/app/components/header/header.component.ts

```diff
@@ -39,4 +43,14 @@ export class HeaderComponent {
           this.headerLinks.set(this.myAccountLink, true);
       }
   }
+
+  login() : void {
+    const dialogConfig = new MatDialogConfig();
+
+    dialogConfig.disableClose = true;
```

Comment:
Is this line disabling the ability to close the dialog by clicking outside of the dialog? If so, do we want that? @PinkyPotrick